UMLEditor/UMLEditor/Classes/Diagram.cs

```diff
@@ -67,6 +69,32 @@ private bool IsClassInRelationship(string name)
         }
         return false;
     }
+    /// <summary>
+    /// Checks if the current diagram has the given class
+    /// </summary>
+    /// <param name="className">The name of the class to check</param>
+    /// <exception cref="ClassNonexistentException"></exception>
+    private void HasClass(string className)
```

Comment:
The tag for the exception does not have a body

I don't like the way this method is named. "HasClass" sounds like it should return a boolean that is false when the class does not exist, and would then be used like _if (HasClass(thing)_. I wanted to use a name like _AssertHasClass_ or _EnsureHasClass_ because those names indicate that the class not existing is an exceptional situation/ error that would cause an exception to be raised.